api/tests/tooling/server/http-test-server.js

```diff
@@ -1,7 +1,8 @@
 const Hapi = require('@hapi/hapi');
+const preResponseUtils = require('../../../lib/infrastructure/utils/pre-response-utils');
 
 /**
- * ⚠️ You must declare your stubs before calling the HttpTestServer constructor (because of Node.Js memoization).
+ * ⚠️ You must declare your stubs before calling the HttpTestServer constructor (because of Node.Js memorization).
```

Comment:
Je pense que l'ancien terme était le bon: https://scotch.io/tutorials/understanding-memoization-in-javascript

Reply:
Done :)